src/components/Tabs/Tabs.test.js

```diff
@@ -0,0 +1,194 @@
+          </Tab>
+          <Tab title="Tab two" id="t-two">
+            Tab two content
+          </Tab>
+          <Tab title="Tab three" id="t-three">
+            Tab two content
+          </Tab>
+        </Tabs>
+      );
+    })
+
+    // it('Should focus the first tab by default', () => {
+    //   cut
+    //   .find('button#t-one-tab')
+    //   .simulate('click');
+    //
+    //   const focusedElement = document.activeElement;
+    //
+    //   expect(cut
+    //     .find('button#t-one-tab')
+    //     .matchesElement(focusedElement));
+    // });
+    //
+    // // Test for default switch/fall-through switch statement
+    // it('Should not do anything if random keys are pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-one-tab',
+    //     end: 'button#t-one-tab',
+    //     key: 84 // "t" key should not do anything
+    //   });
+    // });
+    //
+    // it('Should focus the next tab when the right arrow key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-one-tab',
+    //     end: 'button#t-two-tab',
+    //     key: 39
+    //   })
+    // });
+    //
+    // it('Should focus the next tab when the down arrow key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-one-tab',
+    //     end: 'button#t-two-tab',
+    //     key: 40
+    //   });
+    // });
+    //
+    // it('Should focus the previous tab when the left arrow key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-two-tab',
+    //     end: 'button#t-one-tab',
+    //     key: 37
+    //   });
+    // });
+    //
+    // it('Should focus the previous tab when the up arrow key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-two-tab',
+    //     end: 'button#t-one-tab',
+    //     key: 38
+    //   });
+    // });
+    //
+    // it('Should focus the last tab when the End key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-one-tab',
+    //     end: 'button#t-three-tab',
+    //     key: 35
+    //   });
+    // });
+    //
+    // it('Should focus the first tab when the Home key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-three-tab',
+    //     end: 'button#t-one-tab',
+    //     key: 36
+    //   });
+    // });
+    //
+    // // Right arrow key should focus first tab if last tab has focus
+    // it('Should focus the first tab when last tab has focus and the right key is pressed', () => {
+    //   simulateKeyboardInteraction({
+    //     component: cut,
+    //     start: 'button#t-three-tab',
+    //     end: 'button#t-one-tab',
+    //     key: 39
+    //   });
+    // });
+
```

Comment:
Oooh, these should be uncommented and fixed if necessary

Reply:
These are now uncommented and all pass.